packages/mobile/src/screens/profile-screen/ExpandableBio.tsx

```diff
@@ -83,13 +84,15 @@ export const ExpandableBio = ({ profile }: ExpandableBioProps) => {
     <View style={styles.root}>
       <View>
         {bio ? (
-          <Text
-            numberOfLines={fullBioHeight && !isExpanded ? 2 : 0}
-            style={styles.bio}
-            onLayout={handleBioLayout}
-          >
-            {bio}
-          </Text>
+          <HyperLink>
```

Comment:
Does this not need a prop to handle the link press?

Reply:
updated!